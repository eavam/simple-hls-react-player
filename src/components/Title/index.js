import React from 'react'
import styled from 'styled-components'
import PropTypes from 'prop-types'

const Title = ({ title }) =>
  <Root>
    {title}
  </Root>

Title.propTypes = {
  title: PropTypes.string
}

const Root = styled.div`
  color: #fff;
  font-size: 30px;
`

export default Title
