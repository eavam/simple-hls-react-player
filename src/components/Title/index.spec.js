import React from 'react'
import renderer from 'react-test-renderer'
import { mount } from 'enzyme'
import Title from './index.js'

test('Title', () => {

  const props = {
    title: 'Test'
  }

  const enzymeWrapper = mount(<Title {...props} />)

  expect(enzymeWrapper.find('.title').text()).toBe('Test')

  const componentDisabled = renderer.create(
    <Title {...props} />
  )
  let tree = componentDisabled.toJSON()
  expect(tree).toMatchSnapshot()

})