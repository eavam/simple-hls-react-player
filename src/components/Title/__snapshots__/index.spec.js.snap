// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title 1`] = `
<div
  className="css-rh87r1"
>
  Test
</div>
`;
